src/meta/src/error.rs

```diff
@@ -94,6 +101,14 @@ impl MetaError {
         use std::borrow::Borrow;
         std::matches!(self.inner.borrow(), &MetaErrorInner::InvalidWorker(_))
     }
+
+    pub fn catalog_not_found<T: ToString>(relation: &'static str, name: T) -> Self {
```

Comment:
`ToString` calls `Display::fmt`. I think what we need is `Into<String>` in most cases.

```suggestion
    pub fn catalog_not_found<T: Into<String>>(relation: &'static str, name: T) -> Self {
```